utils/jwt.ts

```diff
@@ -18,10 +18,27 @@ export function validateJwt(req: Request, res: Response, next: NextFunction) {
   jwt.verify(token, process.env.TOKEN_SECRET as string, (err: VerifyErrors, decoded: any) => {
     if (err) {
       logger.error(err);
-      return res.sendStatus(403).end();
+      return res.status(403).json("refresh token is expired").end();
     }
 
     req.body.id = decoded.id;
     next();
   });
 }
+
+export function validateLobbyJwt(req: Request, res: Response, next: NextFunction) {
```

Comment:
i like ur idea of having the validators be merged together as one function! let's see if there is a good way to do it!

Reply:
I merged them into one function. Can you double check if what I did with passing `next` around is valid?